spec.html

```diff
@@ -125,6 +125,61 @@ <h1>Updated Productions</h1>
   </emu-clause>
 </emu-clause>
 
+<emu-clause id="sec-new-ecmascript-specification-types">
+  <h1>New ECMAScript Specification Types</h1>
+
+  <emu-clause id="sec-classes-specification-types">
+    <h1>Classes Specification Types</h1>
+
+    <emu-clause id="sec-elementdescriptor-specification-type" aoid=ElementDescriptor>
+      <h1>The ElementDescriptor Specification Type</h1>
```

Comment:
Most specification types don't use PascalCase, but are space separated. So this would be `Element Descriptor` rather than `ElementDescriptor`. See https://tc39.github.io/ecma262/#sec-property-descriptor-specification-type as an example. While this isn't always the case, I think `Property Descriptor` should be our guiding example here.

Reply:
Ugh, I am actually really not a fan of that spec convention, or how the spec uses `Syntax Error`. It's used really inconsistently, for example when you have an actual Record, it's denoted without the space. I'm happy with the wording here; we can spend more time bikeshedding when merging into the main spec.